src/ssb_project_cli/ssb_project/app.py

```diff
@@ -251,31 +286,38 @@ def create(
     if " " in project_name:
         raise ValueError("Spaces not allowed in projectname, use underscore?")
 
-    if not skip_github and not github_token:
+    if add_github and not github_token:
         raise ValueError("Needs GitHub token, please specify with --github-token")
 
     if not debug_without_create_repo:
-        if not skip_github and is_github_repo(github_token, project_name):
+        if add_github and is_github_repo(github_token, project_name):
             raise ValueError(f"The repo {project_name} already exists on GitHub.")
 
-    create_project_from_template(project_name, description)
+    if add_github and description == "":
+        description = request_project_description()
 
+    create_project_from_template(project_name, description)
     # Create empty folder on root
     # Get content from template to local
-    # git init ?
-    # git add ?
-
-    # 2. Create github repo
-    if not skip_github:
+    # git init
+    # git add
+    # git initial commit
+
+    # Wont push poetry.lock, poetry install not called yet
+    git_repo_dir = DEFAULT_REPO_CREATE_PATH.joinpath(project_name)
+    if add_github:
+        # 2. Create github repo
         print("Initialise empty repo on Github")
         repo_url = create_github(github_token, project_name, repo_privacy, description)
 
         print("Create local repo and push to Github")
-        git_repo_dir = DEFAULT_REPO_CREATE_PATH.joinpath(project_name)
         make_git_repo_and_push(github_token, repo_url, git_repo_dir)
 
         print("Set branch protection rules.")
         set_branch_protection_rules(github_token, project_name)
+    else:
+        # Wont commit poetry.lock, poetry install not called yet
+        make_and_init_git_repo(git_repo_dir)
 
     # 4. Add metadata about creation
```

Comment:
We can remove all of this. If we really need the code in the future we can get it from the git history

Reply:
^